config/extension_import_files.go

```diff
@@ -40,7 +41,8 @@ func (item *ImportFiles) normalize() {
 	}
 
 	if item.Prefix == nil {
-		prefix := util.EncodeBase64Sha1(item.Name) + "_"
+		reg := regexp.MustCompile("[^a-zA-Z]+")
+		prefix := reg.ReplaceAllString(util.EncodeBase64Sha1(item.Name), "") + "_"
```

Comment:
I suppose the prefix does not need to always be the same length?

Reply:
It does not. It's only there really to avoid file conflicts. It wasn't always the same length before this change anyways